domains/hk.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ericsia",
+        "email": "medlexopm@gmail.com"
+    },
+    "record": {
+        "A": "185.27.134.137"
```

IP should be in an array
```suggestion
        "A": ["185.27.134.137"]
```